pkgs/development/python-modules/pytest-xdist/default.nix

```
@@ -66,11 +64,11 @@ buildPythonPackage rec {
 
   setupHook = ./setup-hook.sh;
 
-  meta = with lib; {
-    changelog = "https://github.com/pytest-dev/pytest-xdist/blob/v${version}/CHANGELOG.rst";
-    description = "Pytest xdist plugin for distributed testing and loop-on-failing modes";
+  meta = {
+    changelog = "https://github.com/pytest-dev/pytest-xdist/blob/${src.tag}/CHANGELOG.rst";
+    description = "Pytest plugin for distributed testing, most importantly across multiple CPUs";
```

Not blocking, but not so sure I love the wording here. Maybe:
```suggestion
    description = "Pytest plugin for distributed (including multi-core) testing and loop-on-failing modes";
```